spec/integration/auto_polling_spec.cr

```diff
@@ -0,0 +1,42 @@
+require "../spec_helper"
+require "file_utils"
+
+describe "Auto polling" do
```

Comment:
suggestion: add a test for disabling auto polling when passing 0 as the `poll_interval`